ui/pages/settings/advanced-tab/advanced-tab.component.js

```diff
@@ -601,21 +601,41 @@ export default class AdvancedTab extends PureComponent {
     );
   }
 
-  handleLockChange(time) {
+  handleLockChange(autoLockTimeLimitBeforeNormalization) {
     const { t } = this.context;
-    const autoLockTimeLimit = Math.max(Number(time), 0);
 
-    this.setState(() => {
-      let lockTimeError = '';
+    if (autoLockTimeLimitBeforeNormalization === '') {
+      this.setState({
+        autoLockTimeLimitBeforeNormalization,
+        autoLockTimeLimit: '0',
```

Comment:
Nit: The default auto lock time limit is now set in 3 different places (the parent component, here, and the placeholder). Perhaps we could introduce a constant to ensure they stay in sync?

Reply:
Hmm you're right. A constant would be good. Since you've approved I'll wait to see if any other changes are necessary and either add that here or in a different PR.

Reply:
Okay, references consolidated here: [2069663](https://github.com/MetaMask/metamask-extension/pull/19653/commits/20696634cf5de72b2674f6d0388453441e23051a)